packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -116,7 +118,7 @@ export class ColorPicker
     }
   }
 
-  /** When true, hides the RGB/HSV channel inputs */
+  /** When `true`, hides the RGB/HSV channel inputs */
```

Comment:
Awesome! It looks like we could add in the period at the end of this line as well?